src/test/scala/io/viash/helpers/IOTest.scala

```diff
@@ -94,4 +95,39 @@ class IOTest extends AnyFunSuite with BeforeAndAfter {
     val result = Try(IO.resolveProjectPath("/test/test.txt", None))
     assert(result.isFailure)
   }
+
+  test("anonymizePath with a common path") {
+    val basePath = Some(Paths.get("/foo/bar"))
+    val path = "/foo/bar/baz/test.txt"
+    val result = IO.anonymizePath(basePath, path)
+    assert(result == "baz/test.txt")
+  }
+
+  test("anonymizePath with a different base path") {
+    val basePath = Some(Paths.get("/foo/bar"))
+    val path = "/foo/baz/test.txt"
+    val result = IO.anonymizePath(basePath, path)
+    assert(result == "[anonymized]/test.txt")
+  }
+
+  test("anynimizePath with a completely different base path") {
+    val basePath = Some(Paths.get("/foo/bar"))
+    val path = "/tmp/foo/bar/baz/test.txt"
+    val result = IO.anonymizePath(basePath, path)
+    assert(result == "[anonymized]/test.txt")
+  }
+
+  test("anonymizePath with a common path and a trailing slash") {
+    val basePath = Some(Paths.get("/foo/bar"))
+    val path = "/foo/bar/baz/test.txt"
+    val result = IO.anonymizePath(basePath, path)
+    assert(result == "baz/test.txt")
+  }
```

Comment:
There is no trailing slash in this example, or am I mistaken?

```suggestion
  test("anonymizePath with a common path and a trailing slash") {
    val basePath = Some(Paths.get("/foo/bar/"))
    val path = "/foo/bar/baz/test.txt"
    val result = IO.anonymizePath(basePath, path)
    assert(result == "baz/test.txt")
  }
```

Or


```suggestion
  test("anonymizePath with a common path and a trailing slash") {
    val basePath = Some(Paths.get("/foo/bar"))
    val path = "/foo/bar/baz/test_folder/"
    val result = IO.anonymizePath(basePath, path)
    assert(result == "baz/test_folder/")
  }
```

Reply:
Thanks for spotting that! A refactor went wrong. :stuck_out_tongue: